index.html

```diff
@@ -30,14 +30,74 @@ <h1 class="info_box">
     </div>
     <div>
         <p class="contact_section"><a href="#">LET'S CONNECT</a></p>
+        
     </div>
+    <section>
         <ul>
             <li><a href="#"><img src="cat.svg" width="1" height="15" alt="cat"></a></li>
             <li><a href="#"><img src="bird.svg" width="1" height="15" alt="bird"></a></li>
             <li><a href="#"><img src="m.svg" width="1" height="15" alt="M"></a></li>
             <li><a href="#"><img src="n.png" width="1" height="15" alt="N"></a></li>
             <li><a href="#"><img src="two.svg" width="1" height="15" alt="two"></a></li>
-         </ul>
-    <h3><a href="#">LET'S GET INTERACTIVE</a></h3>
+        </ul>
+        <h3><a href="#">LET'S GET INTERACTIVE</a></h3>
+    </section>
+
+        
+    <section class="section_works_container">
+       <header>
+            <img class="sec_in" src="snap.svg" alt="selfie">
+            <div class="tonic">
+                <h2>Tonic</h2>
+            </div>
+            <div>
+                <ul class="sec_h">
+                    <li><a href="#"><img src="canopy.svg" alt="canopy"></a></li>
+                    <li><a href="#"><img src="Count.svg" width="1" height="5" alt="count"></a></li>
+                    <li><a href="#"><img src="bed.svg" height="22" alt="bed"></a></li>
+                    <li><a href="#"><img src="cc.svg" width="1" height="5" alt="dot"></a></li>
+                    <li><a href="#"><img src="time.svg" alt="date"></a></li>
+                </ul>
+            </div>
+       </header>
+        <section>
+            <h3 class="p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h3>
+            <div>
+                <ul class="push">
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">CSS</a></li>
+                    <li><a href="#">Javascript</a></li>
+                </ul>
+            </div>
+            <h3 class="endpush"><a href="#">See Project</a></h3> 
+        </section>
```

Comment:
Please stick to the Figma design provided as much as possible and also add three more works here and make them look like the given design here I put your design along with the Figma one.

| Your Design | Figma Design |
|--------------|---------------|
| <img width="303" alt="image" src="https://user-images.githubusercontent.com/34578631/170740156-3acfe002-8550-49e4-b9dd-2f0658da5667.png"> | <img width="373" alt="image" src="https://user-images.githubusercontent.com/34578631/170740113-1fa53666-9da4-46ea-bcb3-c6169d238491.png"> |